library/ccm.c

```diff
@@ -49,6 +49,13 @@
 #endif /* MBEDTLS_PLATFORM_C */
 #endif /* MBEDTLS_SELF_TEST && MBEDTLS_AES_C */
 
+#if defined(MBEDTLS_CCM_ALT)
+#define CCM_DEFAULT_INLINE
```

Comment:
Even if this is a “private” identifier used only in an implementation, it's global, so I'd feel safer to call it `MBEDTLS_CCM_DEFAULT_INLINE`.